manifests/volume.pp

```diff
@@ -64,7 +64,7 @@
 define lvm::volume (
   Enum['present', 'absent', 'cleaned'] $ensure,
   Stdlib::Absolutepath $pv,
-  Stdlib::Absolutepath $vg,
+  String[1] $vg,
```

Comment:
I suggest to add a unit test here that explicitly validates strings, otherwise people might change it back to `Stdlib::Absolutepath` during the next refactoring.

Reply:
Agreed! Currently implementing, this should not have got past testing :)